CHANGELOG.md

```diff
@@ -1,5 +1,294 @@
 # Change Log
 
+## [0.7.0](https://github.com/confluentinc/ksql/releases/tag/v0.7.0-ksqldb) (2020-02-11)
+
+### Features
+
+* add a new default SchemaRegistryClient and remove default for SR url ([#4325](https://github.com/confluentinc/ksql/pull/4325)) ([e045f7c](https://github.com/confluentinc/ksql/commit/e045f7c))
```

Comment:
Can we add primitive keys please : https://github.com/confluentinc/ksql/pull/4478

Reply:
Added manually. The reason it didn't show up in the autogenerated changelog is because the commit message is "Docs for primitive key support (#4478)" which does not follow the conventional commits pattern.